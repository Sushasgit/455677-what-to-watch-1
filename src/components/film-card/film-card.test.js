import React from 'react';
import renderer from 'react-test-renderer';
import FilmCard from './film-card.jsx';

it(`FilmCard run correctly`, () => {
  const tree = renderer
    .create(<FilmCard
      film={{}}
      onClick={jest.fn()}
      isPlaying={true}
      handleActiveCard={jest.fn()}
      clearActiveCard={jest.fn()}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
