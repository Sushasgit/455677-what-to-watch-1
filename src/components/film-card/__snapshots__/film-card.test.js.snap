// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmCard run correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
>
  <button
    className="small-movie-card__play-btn"
    onClick={[Function]}
    type="button"
  >
    Play
  </button>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Fantastic Beasts: The Crimes of Grindelwald"
      height="175"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    />
  </h3>
</article>
`;
